pkg/controller/nginxingress/nginx_ingress_controller.go

```diff
@@ -35,6 +35,12 @@ const (
 	defaultMaxReplicas = 100
 )
 
+const (
+	rapidTargetCpuUtilization   = 55
+	defaultTargetCPUUtilization = 70
```

Comment:
nit: can we rename to balanced? I don’t want to couple the fact that we have balanced as an option with the fact that we also have a default. Instead, we should have logic that, in the default case, returns balanced (or whatever we want it to) - not define balanced as the default to begin with